src/components/inputs/LocationType.vue

```diff
@@ -0,0 +1,200 @@
+<template>
+  <div ref="rootEl">
+    <button :class="selectedClass" @click="onTriggerClick"></button>
+    <transition name="fade">
+      <ul v-if="isListVisible">
+        <li v-for="option in options" v-bind:key="option.value">
+          <a :class="option.key" alt="option.description" @click="onListItemClick(option.value)">{{
+            option.label
+          }}</a>
+        </li>
+      </ul>
+    </transition>
+    <select v-bind:value="value" @change="onSelectChange" :class="selectedClass">
+      <option v-for="option in options" v-bind:key="option.value" v-bind:value="option.value">
+        {{ option.label }}
+      </option>
+    </select>
+  </div>
+</template>
+<style scoped lang="scss">
+@import "../../style/variables";
+
+$locationTypeIconPaths: (
+  (home, "../../assets/icons/symbol-house.svg"),
+  (work, "../../assets/icons/symbol-work.svg"),
+  (health, "../../assets/icons/symbol-health.svg"),
+  (wellness, "../../assets/icons/symbol-wellness.svg"),
+  (transport, "../../assets/icons/symbol-transport.svg"),
+  (education, "../../assets/icons/symbol-education.svg")
+);
+
+div {
+  position: relative;
+  overflow: visible;
+  width: 28px;
+  height: 28px;
+}
+
+button {
+  display: none;
+
+  @media (min-width: $breakpoint-tablet-portrait) {
+    width: 28px;
+    height: 28px;
+    border: 0 none;
+    background-color: transparent;
+    background-repeat: no-repeat;
+    background-position: left center;
+    padding-left: 32px;
+    cursor: pointer;
+    display: initial;
+  }
+
+  &::after {
+    content: " ";
+    display: block;
+    height: 100%;
+    width: 8px;
+    background: transparent url("../../assets/carret-down.svg") no-repeat center center;
+  }
+
+  @each $key, $path in $locationTypeIconPaths {
+    &.selected-#{$key} {
+      background-image: url($path);
+    }
+  }
+}
+
+ul {
+  display: none;
+
+  @media (min-width: $breakpoint-tablet-portrait) {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    position: absolute;
+    top: 100%;
+    left: 0;
+    margin: 16px 0 0 0;
+    background: white;
+    list-style: none outside;
+    width: 162px;
+    padding: 9px 12px;
+    border-radius: 3px;
+    box-shadow: 0 2px 4px 0 #00000055;
+
+    &.fade-enter-active,
+    &.fade-leave-active {
+      transition: opacity 0.2s ease-in-out, margin-top 0.2s ease-in-out;
+    }
+
+    &.fade-enter,
+    &.fade-leave-to {
+      margin-top: 12px;
+      opacity: 0;
+    }
+  }
+
+  li {
+    margin: 5px 11px;
+
+    a {
+      display: block;
+      width: 30px;
+      padding-top: 30px;
+      background-size: 26px 26px;
+      background-position: center 2px;
+      background-repeat: no-repeat;
+      cursor: pointer;
```

Comment:
Isn't this the default value?

Reply:
Not without an href tag, so far that I know. The cursor would remain in its default state...

Reply:
Replaced with a button tag as discussed, it makes more sense